IDE/IAR-EWARM/Projects/benchmark/wolfCrypt-benchmark.ewp

```diff
@@ -937,7 +937,7 @@
         </option>
         <option>
           <name>IarchiveOutput</name>
-          <state>###Unitialized###</state>
+          <state>###Uninitialized###</state>
```

Comment:
Not sure about this series of fixes: Is `Unitialized` a documented misspelling of EWP files?

Reply:
Indeed, I don't know what these files are, but *.ewp files:
```
$ grep -R '>###Uninitialized###'
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewd:          <state>###Uninitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewd:          <state>###Uninitialized###</state>
$ 
```
differ from *.ewd files:
```
$ grep -R '>###Unitialized###'
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_lib_SAMV71_XULT/wolfcrypt_lib.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/wolfcrypt_test.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/Projects/test/wolfCrypt-test.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/Projects/benchmark/wolfCrypt-benchmark.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewp:          <state>###Unitialized###</state>
IDE/IAR-EWARM/Projects/lib/wolfSSL-Lib.ewp:          <state>###Unitialized###</state>
$ 
```